src/compiler/python_generator.cc

```diff
@@ -200,17 +202,16 @@ bool GetModuleAndMessagePath(const Descriptor* type,
     path_elem_type = path_elem_type->containing_type();
   } while (path_elem_type != nullptr);
   string file_name = type->file()->name();
-  string module_name;
   static const int proto_suffix_length = strlen(".proto");
   if (!(file_name.size() > static_cast<size_t>(proto_suffix_length) &&
         file_name.find_last_of(".proto") == file_name.size() - 1)) {
     return false;
   }
-  module_name = file_name.substr(
+  string module = file_name.substr(
```

Comment:
If you do a direct copy, be sure to leave a TODO in the code and potentially file a feature request with the upstream source of the function asking for it to be exposed and supported. :-)